### ISSUE TITLE
Just want to embed a subtitle but keep video exactly the same, what's the best settings?

### ISSUE THREAD

**Issue by Kusaywa** [RandomEngy/VidCoder#259]
I successfully embedded a subtitle into a video, VidCoder worked great but the video went from 7gig down to 3. They were separate video (mkv) and srt subtitle file. What are the best settings for keeping the video exactly the same but only add the subtitle. Thanks for any help!

**Comment by RandomEngy** [comment]
You would want to use a muxing program. Video conversion is mandatory in VidCoder; it inherits this from the Handbrake engine.

**Comment by Kusaywa** [comment]
Yes. Is there something simple out there? I don't plan on doing it a lot, but I'm interested in doing it to this one video, for now...

**Comment by RandomEngy** [comment]
Haven't used one for a while. I think I used mp4box a while back but I think that's command-line.

**Comment by Barough** [comment]
For muxing in stuff into a MKV container so is my recommendation MKVToolNix.

**Comment by Kusaywa** [comment]
Sorry, meant to say hardcode in my title... Changed it!!! MKVToolnix will let you embed but not hardcode.

**Comment by Barough** [comment]
If u gonna hardcode a subtitle so do u have to re-encode the existing video to get the sub hardcoded. It's as simple as that.
